tutorial.md

```diff
@@ -64,6 +64,30 @@ of the features they encompass.
     abstractions are built on top of Legion as part of the development
     of a conjugate gradient solver.
 
+## Realm API Tutorial
```

Comment:
I think we should move this section under `Legion Manual and API Documentation` to keep all the Legion links together, and because most users will come here looking for Legion documentation (rather than Realm).

Reply:
fixed